app/presenters/ApiPresenter.php

```
@@ -98,9 +104,12 @@ public function processVersion01($apiToken, $actionType, $activityId, $value, $d
 
         if ($user && $activity) {
             try {
-                $values = ['user_id' => $user->id, 'activity_id' => (int) $activityId, 'value' => $value];
+                $values = ['user_id' => $user->id, 'activity_id' => (int) $activityId, 'value' => (int) $value];
                 $values['created_at'] = $values['updated_at'] = $datetime ?: new \DateTime();
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '108', column '82'). (undefined)

(MissingImport)